packages/roosterjs-content-model/lib/editor/createContentModelEditorCore.ts

```diff
@@ -54,10 +55,10 @@ export function promoteToContentModelEditorCore(
         // Only use Content Model shadow edit when reuse model is enabled because it relies on cached model for the original model
         cmCore.api.switchShadowEdit = switchShadowEdit;
     }
-
     cmCore.originalApi.createEditorContext = createEditorContext;
     cmCore.originalApi.createContentModel = createContentModel;
     cmCore.originalApi.setContentModel = setContentModel;
+    cmCore.api.createPasteModel = createPasteModel;
```

Comment:
Let's move this up to be together with other api assignment, and also assign to originalApi